webserver/src/main/java/io/kestra/webserver/controllers/ExecutionController.java

```diff
@@ -123,7 +125,8 @@ public PagedResults<Execution> find(
         @Parameter(description = "A flow id filter") @Nullable @QueryValue String flowId,
         @Parameter(description = "The start datetime") @Nullable @Format("yyyy-MM-dd'T'HH:mm[:ss][.SSS][XXX]") @QueryValue ZonedDateTime startDate,
         @Parameter(description = "The end datetime") @Nullable @Format("yyyy-MM-dd'T'HH:mm[:ss][.SSS][XXX]") @QueryValue ZonedDateTime endDate,
-        @Parameter(description = "A state filter") @Nullable @QueryValue List<State.Type> state
+        @Parameter(description = "A state filter") @Nullable @QueryValue List<State.Type> state,
+        @Parameter(description = "A labels filter") @Nullable @QueryValue List<String> labels
```

Comment:
```suggestion
        @Parameter(description = "A labels filter as a list of 'key:value'") @Nullable @QueryValue List<String> labels
```

Please apply the same change to all occurrences of the `labels` query param

Reply:
Thanks